docs/usage/configuration-options.md

```diff
@@ -674,8 +674,8 @@ This option is evaluated at PR/MR creation time.
 
 <!-- prettier-ignore -->
 !!! note
-    GitLab and Gitea implement draft status by checking if the PR's title starts with certain strings.
-    This means that `draftPR` on GitLab and Gitea are incompatible with the legacy method of triggering Renovate to rebase a PR by renaming the PR to start with `rebase!`.
+    Gitea and GitLab implement draft status by checking if the PR's title starts with certain strings.
+    This means that `draftPR` on Gitea and GitLab are incompatible with the legacy method of triggering Renovate to rebase a PR by renaming the PR to start with `rebase!`.
```

Comment:
### Sentence too long

This sentence is too long and tries to explain too much at once. The key points seem to be:

- Because Gitea and GitLab have a different method of checking if a PR is draft, the `draftPR` feature doesn't work as expected.
- You can't use the legacy method of triggering Renovate by prefixing the PR with the text `rebase!`.

#### Why are we mentioning a "legacy method"

If the method is "legacy" why do we mention it? People can use the Dependency Dashboard to request a rebase, but maybe that doesn't work on the affected platforms?

Maybe we can streamline the sentence by dropping the reference to the legacy method?

Reply:
make a suggestion, otherwise i would like to shift this to a seperate pr. Intension was to order platforms